src/engines/html5/media-source/adapters/fairplay-drm-handler.js

```diff
@@ -114,7 +117,7 @@ class FairplayDrmHandler {
     let message = event.message;
     let request = new XMLHttpRequest();
     request.responseType = 'arraybuffer';
-    request.addEventListener('load', (e: Event) => this._licenseRequestLoaded(e), false);
+    request.addEventListener('load', (e: Event) => this._licenseRequestLoaded(e));
```

Comment:
Why not use event manager also here?

Reply:
Could be many requests and after it's finish the request object is not relevant anymore, it's a local var.
It'll fill the event manager with non relevant handlers

Reply:
I don't understand, like the way it is now the listener will be never removed...

Reply:
Right, it should be removed.